Notely/lib/screens/home/home.dart

```diff
@@ -0,0 +1,196 @@
+import 'package:Notely/models/user.dart';
+import 'package:Notely/services/auth.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_search_bar/flutter_search_bar.dart';
+import 'package:image_picker/image_picker.dart';
+import 'dart:io';
+import 'package:Notely/screens/settings.dart';
+import 'package:Notely/screens/favorites.dart';
+import 'package:provider/provider.dart';
+import 'package:Notely/screens/profile.dart';
+
+class MyHomePage extends StatefulWidget {
+  MyHomePage({Key key, this.title}) : super(key: key);
+
+  final String title; //We should change this to hold our logo image file
+
+  @override
+  _MyHomePageState createState() => _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  dynamic userInfo;
+
+  SearchBar searchBar; //Search bar object
+
+  final AuthService _auth = AuthService();
+
+  File
+      _image; //Will be used to store and reference the image taken or chosen by the user.
+
+  //Constructor for this instantiates searchBar
+  _MyHomePageState() {
+    searchBar = SearchBar(
+        inBar: false,
+        setState: setState,
+        onSubmitted: print,
+        buildDefaultAppBar: buildAppBar);
+  }
+
+  //Function that builds the appbar and returns it.
+  AppBar buildAppBar(BuildContext context) {
+    userInfo = Provider.of<User>(context);
+    //This should contain the searchbar and hamburger button i think?
+    return new AppBar(
+      title: new Text('Notely',
+          style: TextStyle(
+            fontFamily: 'MontserratAlternates',
+          )),
+      actions: [searchBar.getSearchAction(context)],
+    );
+  }
+
+  //Called when the user hits the plus button. Calls showPicker which opens up the modul on the bottom of the screen
+  void _openCamera() {
+    _showPicker(context);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: searchBar.build(context),
+      body: ListView
+          .separated //Creates a separated list of tiles to appear on the main page
+          (
+        itemBuilder: (BuildContext context,
+                int index) //Creates items to appear in the list
+            {
+          return ListTile //This widget will probably be changed to something else, or at least given a clicking ability
+              (title: Text('item $index') //Displays the number on the tile
+                  );
+        },
+        separatorBuilder: (BuildContext context, int index) =>
+            Divider(), //Creates divides between items, will probably be changed to spaces instead of lines later
+        itemCount: 40, //This will change to an infinite scroll later
```

Comment:
The ListView is sometimes sluggish when we load so many things. I think we need to reduce the amount it loads at any one time.